components/browser_kit/introduction.rst

```diff
@@ -0,0 +1,145 @@
+.. index::
+   single: BrowserKit
+   single: Components; BrowserKit
+
+The BrowserKit Component
+========================
+
+    The BrowserKit component simulates the behavior of a web browser.
+
+The BrowserKit component allows you to make web request, click on links and submit forms. 
+
+Installation
+------------
+
+You can install the component in 2 different ways:
+
+* :doc:`Install it via Composer </components/using_components>` (``symfony/browser-kit`` on `Packagist`_);
+* Use the official Git repository (https://github.com/symfony/BrowserKit).
+
+Basic Usage
+-----------
+
+.. note::
+    The component only provides an abstract client and does not provide any "default" backend for the HTTP layer.
+
+Creating a Client
+-----------------
+
+To create your own client you must extend the abstract client class and implement the doRequest method.
+This method accepts a request and should return a response.
+
+.. code-block:: php
+
+    namespace ACME;
```

Comment:
No, we are in the components section here. It should not use framework things like bundles